Updated Example Added for consultation and pharmacy

emmayank opened the pull request.

ravi-prakash-v commented:
@emmayank please resolve the conflicts occured due to previous PR by @shenoyninad and resubmit the PR. 

Also, the descriptions in dhp.yaml need to be corrected. There is no such thing as DHP provider platform or DHP Gateway. Please reset the terminology to include only beckn core spec descriptions. 

The only updates to be made are ennumerations, tags, and examples. Rest will remain the same.

emmayank commented:
Hi @ravi-prakash-v 

following changes have been made.

1. Resolved Merge Conflict
2. Added Submodule Beckn1.1
3. Updated dhp.yml - replaced DHP specific terminology with the beckn protocol terminology.


Thanks
emmayank
